scripts/apps/search/directives/ItemList.js

```diff
@@ -808,9 +811,13 @@ export function ItemList(
             });
 
             var ListItemInfo = function(props) {
+                var itemInfoClassName = 'item-info';
+                if (config.reducedItemListRowHeight) {
+                    itemInfoClassName = 'item-info item-info-reduced-rowheight';
+                }
                 return React.createElement(
                     'div',
-                    {className: 'item-info'},
+                    {className: itemInfoClassName},
```

Comment:
Sure. There already was a classNames import in the file, so I have used the same. Thanks.